src/components/email-flow/EmailFlowProof.tsx

```diff
@@ -1,15 +1,15 @@
-import { BodyText, HeaderText } from 'components/Text'
+import { BodyText, HeaderText } from 'components/ui/Text'
 import BadgeBlock from 'components/badges/BadgeBlock'
-import EmailProof from 'helpers/EmailProof'
 import MintedToken from 'models/MintedToken'
-import ReadyEmailProof from 'components/proofs/ReadyEmailProof'
+import Proof from 'components/proofs/Proof'
+import ProofModel from 'models/Proof'
 
 export default function ({
   proof,
   onMinted,
   onMintFailed,
 }: {
-  proof: EmailProof
+  proof: ProofModel
```

Comment:
Same interface as [here](https://github.com/BigWhaleLabs/seal-cred-frontend/pull/452/files#diff-5b1516035d380825740282f22b8ac9801995a54f0ccbba08ab46bd23de3c8bdbR73)

Reply:
I'm not sure if this should be put in a separate interface